### ISSUE TITLE
Add the family composition to the other family members

### ISSUE THREAD

**Issue by phuongmy** [kids-first/kf-es-model#13]
Currently the family composition value is applied to the child.
Example, when you filter on family_composition = complete_trio --> you only get files of the child. Files of the mother and father are excluded from the file repo.
We should populate this family composition for the parents as well.

Use case 1: On the file repo, if you select family composition is complete_tri, you should get the child's, father's and mother's files.

Use case 2: let's imagine the father has a parent as well --> This father as a participant has 2 family compositions: complete_trio + partial_trio.

Addition note: we'll keep track of the family relations within each family composition.

**Comment by junjun-zhang** [comment]
The effective goal is to assign same `family_composition_type` to all family members involved in the same composition. `family_composition_type` currently can be different depending on the perspective of an individual family member. The gist of the change is as follow:

Now
```
participant
   |__ family
          |__ family_data
          |__ family_members
```
After
```
participant
   |__ family
          |__ family_compositions
                |__ family_data
                |__ family_members  // all family member in this composition (eg. complete_trio) have the same  value for composition type
```
Full details will be described here: https://kidsfirst.atlassian.net/wiki/spaces/KFDRC/pages/462553299/Elasticsearch+data+model+and+ETL

**Comment by junjun-zhang** [comment]
addressed by #15
